src/core/tsi/ssl/session_cache/ssl_session_cache.cc

```diff
@@ -94,6 +98,7 @@ SslSessionLRUCache::Node* SslSessionLRUCache::FindLocked(
 }
 
 void SslSessionLRUCache::Put(const char* key, SslSessionPtr session) {
+  if (session == nullptr) return;
```

Comment:
Should we do any error logging here - seems this would be a pretty unexpected scenario?

Reply:
Done.